extensions/wikia/CoppaTool/CoppaToolSpecialController.class.php

```diff
@@ -12,7 +12,8 @@ public function __construct() {
 
 	public function index() {
 		global $wgCentralWikiId;
+		$target = $this->getVal( 'username', $this->getPar() );
 		$title = GlobalTitle::newFromText( 'CoppaTool', NS_SPECIAL, $wgCentralWikiId );
-		$this->getOutput()->redirect( $title->getFullURL() );
+		$this->getOutput()->redirect( $title->getFullURL( [ 'usernameOrIp' => $target ] ) );
```

Comment:
Shouldn't this be 'target' query param?
```suggestion
		$this->getOutput()->redirect( $title->getFullURL( [ 'target' => $target ] ) );
```

Reply:
Ah it's CoppaTool, thanks @mmeller-wikia.